addons/website/static/src/css/website.css

```diff
@@ -974,3 +974,7 @@ div.carousel .item.text_only .container > .carousel-image {
 #oe_main_menu_navbar .navbar-nav.navbar-right:last-child {
   margin-right: 0 !important;
 }
+
+.container nav#oe_main_menu_navbar {
```

Comment:
The screencast display the reason for the change.

![screenshot from 2015 04 29 19 02 45](https://cloud.githubusercontent.com/assets/5112424/7392030/ad6b2d6a-eea2-11e4-9909-8f01380901fd.png)